source/board/override_vbluno51/IO_Config_Override.h

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file    IO_Config_Override.c
+ * @brief   Alternative IO for LPC11U35 based Hardware Interface Circuit
+ *
+ * DAPLink Interface Firmware
+ * Copyright (c) 2009-2017, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __IO_CONFIG_H__
+#define __IO_CONFIG_H__
+
+#include "LPC11Uxx.h"
+#include "daplink.h"
+
+// This GPIO configuration is only valid for the LPC11U35 HIC
+COMPILER_ASSERT(DAPLINK_HIC_ID == DAPLINK_HIC_ID_LPC11U35);
+
+// Peripheral register bit masks (used for pin inits)
+#define FUNC_0                          0
+#define FUNC_1                          1
+#define PULL_DOWN_ENABLED               (1 << 3)
+#define PULL_UP_ENABLED                 (2 << 3)
+#define OPENDRAIN                       (1 << 10)
```

Comment:
Are you unable to use the standard pinout for your board? If not, can you document in this file which pins differ?